test/common/utxobased/db/NewProcessor.spec.ts

```diff
@@ -0,0 +1,322 @@
+/* eslint-disable @typescript-eslint/no-unused-expressions */
+import * as chai from 'chai'
+import { expect } from 'chai'
+import { makeMemoryDisklet } from 'disklet'
+
+import {
+  makeNewProcessor,
+  NewProcessor
+} from '../../../../src/common/utxobased/db/newProcessor'
+import {
+  IProcessorTransaction,
+  ITransactionInput,
+  ITransactionOutput
+} from '../../../../src/common/utxobased/db/types'
+
+chai.should()
+
+describe('Processor transactions tests', () => {
+  let processor: NewProcessor
```

Comment:
Don't share `processor` state between test if not necessary

Reply:
Yeah, fair point. I'll add arguments where needed.